app/controllers/questions_controller.rb

```diff
@@ -36,8 +40,12 @@ def update
   end
 
   def destroy
-    @question.destroy
-    redirect_to question_path
+    if @question.user == current_user
+      @question.destroy
+      redirect_to questions_path, notice: 'Your question was deleted'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.